packages/react-reconciler/src/ReactFiberSuspenseConfig.js

```diff
@@ -11,11 +11,11 @@ import ReactSharedInternals from 'shared/ReactSharedInternals';
 
 const {ReactCurrentBatchConfig} = ReactSharedInternals;
 
-export type SuspenseConfig = {|
+export type SuspenseConfig = {
   timeoutMs: number,
   busyDelayMs?: number,
   busyMinDurationMs?: number,
-|};
+};
```

Comment:
Seems unrelated but I assume this change was intentional?

Reply:
Ah nope didn't mean to stage that